app/views/posts/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= form_for @post do |form| %>
   <%= form.label :message %>
-  <%= form.text_field :message %>
+  <%= form.text_area :message %>
```

Comment:
Might want to set the size of this text area and also set it to unchangeable size